cmd/set.go

```diff
@@ -31,6 +33,8 @@ var setCmd = &cobra.Command{
 }
 
 func init() {
-	rootCmd.AddCommand(setCmd)
-	setCmd.AddCommand(setEnvironmentSecretCmd)
+	if setEnabled {
+		rootCmd.AddCommand(setCmd)
+		setCmd.AddCommand(setEnvironmentSecretCmd)
```

Comment:
Should there not be a feature flag on setEnvironmentSecretCmd, even though we want it to be enabled? Might be "smør på flesk"...

Reply:
Does it make sense to enable "set" but keep the sub-command "set environment-secret" disabled? I cannot see any other settable resource. "set-context" is its own command. For now I've only implemented feature toggling for lvl1 subcommands.